devtools/conda-envs/test_env.yaml

```
@@ -1,37 +1,19 @@
 name: openmmforcefields-test
-
 channels:
+  - conda-forge/label/openmm_rc
   - conda-forge
-
 dependencies:
-    # Base depends
   - python
   - pip
-
-    # Testing
   - pytest
   - pytest-cov
   - pytest-xdist
   - pytest-randomly
-
-    # Requirements for converted force field installer
-  - openmm >=7.6.0
-
+  - openmm ~=8.1.2rc1
```

```suggestion
  - openmm >=8.1.2
```
I think that should work.